_studio/shared/umc/core/vm/src/vm_event_linux32.c

```diff
@@ -67,19 +68,24 @@ vm_status vm_event_init(vm_event *event, int32_t manual, int32_t state)
 
     event->manual = manual;
     event->state = state ? 1 : 0;
-    res = pthread_cond_init(&event->cond, 0);
+    res = pthread_condattr_init(&cond_attr);
     if (!res)
     {
-        res = pthread_mutex_init(&event->mutex, 0);
-        if (res)
+        pthread_condattr_setclock(&cond_attr, CLOCK_MONOTONIC);
+        res = pthread_cond_init(&event->cond, 0);
```

Comment:
Didn't you miss `res = pthread_cond_init(&event->cond, &cond_attr);`  here?

Reply:
Done. I missed this when I ported from my test environment.

Reply:
Yes indeed. I missed this when I ported from my test environment (Android).